pokemongo_bot/cell_workers/incubate_eggs.py

```diff
@@ -176,6 +180,9 @@ def _check_inventory(self, lookup_ids=[]):
                             pokemon.get('individual_stamina', 0)
                         ]})
                     matched_pokemon.append(pokemon)
+                    #remove as egg and add as pokemon
+                    inventory.pokemons().remove(pokemon['id'])
+                    inventory.pokemons().add(inventory.Pokemon(pokemon))
```

Comment:
Removing the egg and adding the Pokemon can easily be done in the `_hatch_eggs`.
If we remove the api call, the we will need to update the cached inventory with incubators when no uses are left and use distance walked in metrics.